controllers/admin/AdminImportController.php

```diff
@@ -2116,7 +2789,10 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
                     }
 
                     if ($error) {
-                        $this->warnings[] = sprintf($this->trans('Product #%1$d: the picture (%2$s) cannot be saved.', array(), 'Admin.Advparameters.Notification'), $image->id_product, $url);
+                        $this->warnings[] = $this->trans('Product #%1$d: the picture (%2$s) cannot be saved.', array(
```

Comment:
Same comment about translation placeholder. Please use a variable name surrounded by `%%` an add it in the second parameter.